api/projects/tests/test_serializers.py

```diff
@@ -78,9 +81,32 @@ def test_ProjectSerializer_get_migration_status_calls_migrator_with_correct_argu
         (ProjectIdentityMigrationStatus.NOT_APPLICABLE.value, False),
     ],
 )
-def test_ProjectSerializer_get_use_edge_identities(project, migration_status, expected):
+def test_ProjectListSerializer_get_use_edge_identities(
+    project, migration_status, expected
+):
     # Given
-    serializer = ProjectSerializer(context={"migration_status": migration_status})
+    serializer = ProjectListSerializer(context={"migration_status": migration_status})
+
+    # When/Then
+    assert expected is serializer.get_use_edge_identities(project)
+
+
+@pytest.mark.parametrize(
+    "migration_status, expected",
+    [
+        (ProjectIdentityMigrationStatus.MIGRATION_COMPLETED.value, True),
+        (ProjectIdentityMigrationStatus.MIGRATION_IN_PROGRESS.value, False),
+        (ProjectIdentityMigrationStatus.MIGRATION_NOT_STARTED.value, False),
+        (ProjectIdentityMigrationStatus.NOT_APPLICABLE.value, False),
+    ],
+)
+def test_ProjectRetrieveSerializer_get_use_edge_identities(
+    project, migration_status, expected
+):
+    # Given
+    serializer = ProjectRetrieveSerializer(
+        context={"migration_status": migration_status}
+    )
```

Comment:
I'm not sure why Github thinks this is all new code?

Reply:
This is another test